### PR TITLE
EN: Learn.

### PR DESCRIPTION
**Pull request by DabirSA**
Training quest and tutorial stuff.

### CODE REVIEW

File: Tutorial/Window/wh_tl_od_010409037.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name01#0,"""About \""Just Guard\"""""
+text01#0,"""This explains the ""Just Guard"" skill.<br><br>Normally, when Guarding, you still take minor damage."""
```
Comment (alama) [inline Tutorial/Window/wh_tl_od_010409037.csv]: `wh_tl_od_010409037.csv` `text01#0` missing `\""`

Comment (DabirSA) [reply Tutorial/Window/wh_tl_od_010409037.csv]: fixed

File: Tutorial/Dialogue/od_010409037.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+text01#0,"""ご苦労だった。<br>期待を集めるだけあって<br>基礎はしっかりしているな。"""
+text02#0,"""だが私に言わせれば、その程度なら<br>掃いて捨てるほどいる。本当に<br>伸びるかどうかはまったく別の問題だ。"""
+text03#0,"""とはいえ君はまだまだ<br>伸びそうな気がする。"""
+text04#0,"""ところで、スキルのジャストガードは<br>活用しているかな？"""
+11_voice_NPC_5054_100#0,"""ハンターとして生きていくなら<br>習得して損はない。<br>では、また会おう！"""
+text100#0,"""About \""Just Guard\"""
```
Comment (alama) [inline Tutorial/Dialogue/od_010409037.csv]: `od_010409037.csv` `text100` is missing `""`  at the end

Comment (DabirSA) [reply Tutorial/Dialogue/od_010409037.csv]: fixed, screaming

File: Tutorial/Dialogue/co_011014.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+name01#0,"""Telepipes"""
```
Comment (alama) [inline Tutorial/Dialogue/co_011014.csv]: co_011014.csv's lines `linewindow#0` and `text00#0` are missing

Comment (alama) [reply Tutorial/Dialogue/co_011014.csv]: also `name01#0` does not exist in this CSV

Comment (DabirSA) [reply Tutorial/Dialogue/co_011014.csv]: think fixed

**Review by SynthSy — APPROVED** [review]

File: Tutorial/Dialogue/od_010409037.csv (hunk 2 above)
Comment (alama) [reply Tutorial/Dialogue/od_010409037.csv]: line text100 is still odd

**Pull request merged**